pulsar-broker/src/main/java/org/apache/pulsar/PulsarStandaloneBuilder.java

```diff
@@ -97,10 +108,31 @@ public PulsarStandaloneBuilder withAdvertisedAddress(String advertisedAddress) {
         return this;
     }
 
-    public PulsarStandalone build() {
-        ServiceConfiguration config = new ServiceConfiguration();
-        config.setClusterName("standalone");
-        pulsarStandalone.setConfig(config);
+    public PulsarStandalone build()
+    {
+        // Change IOException and ConfigurationException into a RuntimeException, because if the
+        // config file isn't readable, there is nothing a caller can do, so don't bother with
+        // a checked exception that needs to be catched
+        try {
+            // By reading the configuration file here, the user can modify the configurations before
+            // calling PulsarStandalone.start()
+            ServerConfiguration bkServerConf = new ServerConfiguration();
+            bkServerConf.loadConf(new File(pulsarStandalone.getConfigFile()).toURI().toURL());
+            pulsarStandalone.setBkServerConfig(bkServerConf);
+
+            pulsarStandalone.setConfig(PulsarConfigurationLoader.create(
+                    new FileInputStream(pulsarStandalone.getConfigFile()), ServiceConfiguration.class));
+        }
+        catch (IOException | ConfigurationException e) {
```

Comment:
Move catch to previous line after close braces to follow formatting conventions

Reply:
Done.